src/Development/IDE/Types/Location.hs

```diff
@@ -129,6 +130,24 @@ noRange =  Range (Position 0 0) (Position 100000 0)
 showPosition :: Position -> String
 showPosition Position{..} = show (_line + 1) ++ ":" ++ show (_character + 1)
 
+isInsideRange :: Position -> RealSrcSpan -> Bool
+p `isInsideRange` r = sp <= p && p <= ep
+  where (sp, ep) = unpackRealSrcSpan r
+
+-- | Converts from 1-based tuple
+toPos :: (Int,Int) -> Position
+toPos (l,c) = Position (l-1) (c-1)
+
+unpackRealSrcSpan :: GHC.RealSrcSpan -> (Position, Position)
```

Comment:
This seems to be pretty much the same as `Development.IDE.GHC.Error.realSrcSpanToRange`.